docs/tutorials/creating-a-runtime-module.md

```diff
@@ -51,184 +73,286 @@ You should be able to successfully compile this template module with:
 cargo build
 ```
 
-Let's explore some of the things we did for you in this Substrate module template. We will start by looking at the `Cargo.toml` file.
+Let's explore some of the things Substrate module template do for us, starting
+from the `Cargo.toml` file.
+
+## Development
 
 ### Your Module's `std` Feature
 
-In your `Cargo.toml` file, you will notice a few lines about the "`std` feature". In Rust, when you enable `std`, you give your project access to [the Rust standard libraries](https://doc.rust-lang.org/std/). This works just fine when building native binaries.
+In your `my-module/Cargo.toml` file, you will notice a few lines about the
+"`std` feature". In Rust, when you enable `std`, you give your project access to
+[the Rust standard libraries](https://doc.rust-lang.org/std/). This works just
+fine when building native binaries.
 
-However, Substrate also builds the runtime code to Wasm. In this case we use cargo features to disable the Rust standard library when compiling for Wasm because [wasm-unknown-unknown](https://github.com/rust-lang/rust/pull/45905) does not know which allocator to use.
+However, Substrate also builds the runtime code to WebAssembly(Wasm). In this
+case we use cargo features to disable the Rust standard library when compiling
+for Wasm because the compiler does not know which allocator to use. See
```

Comment:
It would be good to get a strong follow up from a Wasm person the precise reasons here.

I dont think it i has to do with the allocator.

Reply:
I thought it was all about binary size